### PR TITLE
[13.0][MIG] website_sale_product_attachment: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 13.0 + tests

@Tecnativa TT27881

### CODE REVIEW

File: website_sale_product_attachment/tests/test_tour.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        product.website_attachment_ids = [(6, 0, [attachment.id])]
+
+    def test_tour(self):
+        tour = "website_sale_product_attachment_tour"
+        self.phantom_js(
+            "/",
+            "odoo.__DEBUG__.services['web_tour.tour'].run('%s')" % tour,
+            "odoo.__DEBUG__.services['web_tour.tour'].tours.%s.ready" % tour,
+            login="admin",
+        )
```
Comment (joao-p-marques) [inline website_sale_product_attachment/tests/test_tour.py]: You could migrate this to use https://github.com/odoo/odoo/blob/13.0/odoo/tests/common.py#L1253
It simplifies the call, and also saves some work when migrating to v14, as `phantom_js` is deprecated.

**Review by chienandalu — APPROVED** [review]
:+1: Tested and code review!

**Comment by pedrobaeza** [comment]
@joao-p-marques can you confirm your review?

File: website_sale_product_attachment/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Instead of configuring attachments per product, to configure products per attachment:
+
+#. You need permissions to access system settings.
+#. Go to *Settings > Dashboard > Activate the developer mode*.
+#. Go to *Settings > Technical > Database Structure > Attachments*.
+#. Pick or create one.
+#. Enable *Attached To > Is public document*.
+#. Use the field *Attached in product templates* to choose the products.
```
Comment (joao-p-marques) [inline website_sale_product_attachment/readme/CONFIGURE.rst]:
```suggestion
#. Use the field *Attached in products* to choose the products.
```

**Review by joao-p-marques — APPROVED** [review]
Functional review ok :+1:

**Pull request merged**